apps/docs/pages/guides/realtime/broadcast.mdx

```diff
@@ -24,46 +24,46 @@ Go to your Supabase project's [API Settings](https://supabase.com/dashboard/proj
   defaultActiveId="js"
   queryGroup="language"
 >
-<TabPanel id="js" label="JavaScript">
+  <TabPanel id="js" label="JavaScript">
```

Comment:
Added these indents in this file as otherwise the prettier didn't pass, and tries to add back ticks on wrong places.